### PR TITLE
User accessibility branch

### PR DESCRIPTION
**Pull request by tacettinkustu**
**Page Title :** 

1. The title has to be more descriptive e.g. Tacettin's Work Portfolio Website.
2. We will add an icon to the page title to make it more distinguish.

**Images Text Alternatives :** No issues found

**Headings :** No issues found

**Color Contrast :** 

1. On the form section there is low contrast between the background and text fields.
2. In the work cards there is low contrast between background and list elements.
3. “My Logo” and “Lets Connect” links need more contrast.

**Resize:**  No issues found, the text does not lose its clarity with the zoom effect.

**Keyboard access and visual focus:** 

1. The only issue is getting the focus to be on the images using the enter key.
2. We have no drop down lists in our site
3. We also have no image links on our site.

**Forms Labels and Errors:**  

1. We will add asterix (*) to emphasis required fields.
2. We have “name” attribute for all form fields. 

**Interaction :** No issues found
 
**Moving content :** No issues found
 
**Multimedia :** No issues found
 
**The basic structure of the page :** No issues found

### CODE REVIEW

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @tacettinkustu 🖐️, 

Good job so far! your portfolio looks good already just a few things I suggest that you work on to go to the next project but you are almost there!



## Required Changes ♻️
- [x] I suggest that you make your navbar fixed, it will be easier for the users to access it whenever they will be on the page
- [x] Navbar links currently are leading to nowhere, I suggest that you link them to the related section

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Giving a name a minimum length in your contact form
- There is much empty space between the first section and navbar especially on desktop and I recognized that this section is inside the header tag please move it out for good use of semantic HTML
👇
sp


### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tacettinkustu** [comment]
> ## Hi @tacettinkustu 🖐️,
> Good job so far! your portfolio looks good already just a few things I suggest that you work on to go to the next project but you are almost there!
> 
> ## Required Changes ♻️
> * [x]  I suggest that you make your navbar fixed, it will be easier for the users to access it whenever they will be on the page
> * [x]  Navbar links currently are leading to nowhere, I suggest that you link them to the related section
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> You can also consider:
> 
> * Giving a name a minimum length in your contact form
> * There is much empty space between the first section and navbar especially on desktop and I recognized that this section is inside the header tag please move it out for good use of semantic HTML
>   👇
>   sp
> 
> ### Cheers and Happy coding!👏👏👏
> Feel free to leave a question, or a comment on the PR thread if something is not clear 100%.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Thank you for your great feedbacks. I fixed required points.

File: index.html [hunk 1]
```diff
@@ -42,19 +46,21 @@
...
+          <a title="menu" href="#" class="hamburger-menu">
+            <i class="fa fa-bars"></i>
+          </a>
+          <div class="nav-menu">
+            <a title="portfolio" href="#works" class="nav-menu-link">Portfolio</a>
+            <a title="about" href="#about-me" class="nav-menu-link">About</a>
+            <a title="contact" href="#contact-with-me" class="nav-menu-link">Contact</a>
+          </div>
+        </nav>
+      </div>
```
Comment (Temmarie) [inline index.html]: - [x]  Good job implementing this required change. There's just a little tweak necessary for you to make. The fixed navbar clashes with the content on the page. The background of the navbar is slightly see-through. Kindly fix this to increase accessibility and overall page aesthetics. 
- [x] Also, consider making the navbar full width of the screen, on the desktop.
_See image below_

image

**Review by Temmarie — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED! :recycle:

### Hi @tacettinkustu and @Munsa1    ! :wave: :wave:

#### You have done really well in implementing the previously required changes.

 However, there are changes required for you to make in order for this project to be approved. 
- - -

### **Required Changes** ♻️
_See the inline comments below for the required changes_


### Optional Changes 👌
Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better.
- N/A

_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

**_"Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."_**
- - - 
**Note that you still have 1 more review as written in the [Code Reviews Limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561), however, if you feel this review is unfair, you can request a second opinion using [this](https://airtable.com/shrQAqnBwek5a0O0s) form.**

__Kindly make the necessary changes and submit another code review request.__

File: index.html (hunk 1 above)
Comment (tacettinkustu) [reply index.html]: Thank you for your feedbacks. I fixed them all.

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi @tacettinkustu , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

# Optional improvements ☑️ 
- The tablet size is the same as the mobile size, I kindly invite you to fix it.
image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**